bin/nvcc_wrapper

```diff
@@ -195,10 +195,10 @@ do
     cuda_args="$cuda_args $1"
     ;;
   #Handle c++11
-  --std=c++1y|-std=c++1y|--std=c++1z|-std=c++1z|--std=gnu++1y|-std=gnu++1y|--std=gnu++1z|-std=gnu++1z)
+  --std=c++1y|-std=c++1y|--std=c++1z|-std=c++1z|--std=gnu++1y|-std=gnu++1y|--std=gnu++1z|-std=gnu++1z|--std=c++2a|-std=c++2a|--std=c++17|-std=c++17)
     fallback_std_flag="-std=c++14"
     # this is hopefully just occurring in a downstream project during CMake feature tests
-    # we really have no choice here but to accept the flag and "promote" to the corresponding C++ standard
+    # we really have no choice here but to accept the flag and change  to an accepted C++ standard
     echo "nvcc_wrapper does not accept intermediate standard flag $1, but it will use $fallback_std_flag instead. It is undefined behavior to use this flag. This should only be occurring during CMake configuration."
```

Comment:
```suggestion
    echo "nvcc_wrapper does not accept standard flag $1 since partial standard flags and C++ standards after C++14 are not supported. $fallback_std_flag is set instead, but using it is undefined behavior. This should only be occurring during CMake configuration."
```

Reply:
Fixed.